repository/SmalltalkCI-Core.package/SCIGemStoneServerConfigSpec.class/properties.json

```diff
@@ -0,0 +1,17 @@
+{
+	"category" : "SmalltalkCI-Core",
```

Comment:
Avoid having all sorts of platform-specific configs in `SmalltalkCI-Core`. I guess this still is reasonable, because it only holds data. And there might be platform-independent configs...you never know 😄